.github/workflows/package-client.yml

```
@@ -331,8 +331,9 @@ jobs:
         # we need to escape the flag another time by adding another `--`.
         run: >-
           npm run electron:release
-          ${{ matrix.platform == 'linux' && '-- appimage' || '' }}
-          ${{ inputs.nightly_build && '-- --nightly' || '' }}
+          ${{ (matrix.platform == 'linux' || inputs.nightly_build) && '--' }}
```

Are spaces added automatically by GitHub?
```suggestion
          ${{ (matrix.platform == 'linux' || inputs.nightly_build) && '-- ' }}
```

---

Not from GitHub but from the yaml spec for [_folded multiline_ (`>`)]

- https://yaml-multiline.info/

[_folded multiline_ (`>`)]: https://yaml.org/spec/1.2.2/#folded-style